Vesper-Agents-Suite/Compound-Collateral-Factor-Agent/src/index.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  Finding, FindingSeverity,
+  FindingType, HandleTransaction,
+  TransactionEvent
+} from 'forta-agent';
+import {
+  FindingGenerator,
+  provideEventCheckerHandler
+} from 'forta-agent-tools';
+import Web3 from 'web3';
+import { COLLATERAL_FACTOR_EVENT_ALERT_ID, COMPOUND_COMPTROLLER_ADDRESS, NEW_COLLATERAL_FACTOR_SIGNATURE } from './utils';
+
+const web3 = new Web3();
+
+const createFindingGenerator = (): FindingGenerator => {
```

Comment:
I think that you can change this a little.

Right now you have a function that return another function, but the outer one doesn't receive any parameter and doesn't do any changes in the inner one. Then you can keep just the inner function.

The other option some parameters to the outer one. We usually pass the `alertId` as a parameter when we create a function like this, in your case it would be something like:
```typescript
const findingGenerator = createFindingGenerator(COLLATERAL_FACTOR_EVENT_ALERT_ID);
```